health/health.c

```diff
@@ -155,7 +155,6 @@ void health_reload_host(RRDHOST *host) {
  * Reload the host configuration for all hosts.
  */
 void health_reload(void) {
-
     rrd_rdlock();
```

Comment:
Add that empty line back

Reply:
I returned it with another commit, but it will be necessary a PR to remove it, because this function has a pattern with this blank like that is not present in other functions of the file.